ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/ShopperInsightsClient.kt

```diff
@@ -161,20 +160,28 @@ class ShopperInsightsClient internal constructor(
     /**
      * Call this method when the PayPal button has been successfully displayed to the buyer.
      * This method sends analytics to help improve the Shopper Insights feature experience.
-     *
-     * @param experiment optional JSON string representing an experiment you want to run
+     * @param presentmentDetails optional JSON string representing an experiment you want to run.
      * @param paymentMethodsDisplayed optional The list of available payment methods,
      * rendered in the same order in which they are displayed
+     * @param buttonType optional Represents the tapped button type.
+     * @param buttonOrder optional Represents this buttons order in context of other buttons.
+     * @param pageType optional Represents the page or view the button is rendered on.
      */
-    fun sendPayPalPresentedEvent(
-        experiment: String? = null,
-        paymentMethodsDisplayed: List<String> = emptyList()
+    fun sendPresentedEvent(
```

Comment:
Rather than passing all of these params to these methods we should move the params to the `PresentmentDetails` object and just pass `buttonType` and `presentmentDetails` in the method signatures for these methods